admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
+for your Web server. If you do not install a memcache you will see a warning on 
+your ownCloud admin page. (A memcache is not required and you may ignore the warning if you prefer.)
+
+A PHP opcache stores compiled PHP scripts so they don't need to be re-compiled 
```

Comment:
opcache === opcode cache, but every resource refers to it as 'opcache' so leave that